goodreads_scraping/parsing/AuthorNamesParser.py

```diff
@@ -0,0 +1,31 @@
+from lxml import etree
+
+from .BaseParser import BaseParser
+
+
+class AuthorNamesParser(BaseParser):
+    def __init__(self, html):
+        self.xml = etree.fromstring(html)
+        self.parsed_author_names = self._parse_names()
+
+    @property
```

Comment:
Is there any reason why you added this?

Reply:
XPath string generated by get_xpath is a static value that does not change based on the object's state or any other external factors, hence imo it makes sence to provide it as a property (similar to atribute).